bokehjs/src/lib/core/ui_events.ts

```diff
@@ -105,7 +105,7 @@ export class UIEvents implements EventListenerObject {
   readonly keydown:      UISignal<KeyEvent>     = new Signal(this, 'keydown')
   readonly keyup:        UISignal<KeyEvent>     = new Signal(this, 'keyup')
 
-  private readonly hammer = new Hammer(this.hit_area, {touchAction: 'auto'})
+  private readonly hammer = new Hammer(this.hit_area, {touchAction: 'auto', inputClass: Hammer.TouchMouseInput})
```

Comment:
Can you add a link to the issue, so that we know why that argument is there? Like this:
```ts
private readonly hammer = new Hammer(this.hit_area, {
  touchAction: 'auto',
  inputClass: Hammer.TouchMouseInput, // https://github.com/bokeh/bokeh/issues/9187
})
```